tools/test-proxy/Azure.Sdk.Tools.TestProxy.Tests/TestHelpers.cs

```diff
@@ -254,8 +253,24 @@ public static bool IncrementFileVersion(string testFolder, string fileName)
             {
                 File.WriteAllText(fullFileName, (++intVersion).ToString());
             }
+        }
 
-            return false;
+        /// <summary>
+        /// Create a new file with an initial version of 1
+        /// </summary>
+        /// <param name="testFolder">The temporary test folder created by TestHelpers.DescribeTestFolder</param>
+        /// <param name="fileName">The file to be created</param>
+        public static void CreateFileWithInitialVersion(string testFolder, string fileName)
+        {
+            string fullFileName = Path.Combine(testFolder, fileName);
+
+            if (File.Exists(fullFileName))
+            {
+                string errorString = String.Format("FileName {0} already exists", fullFileName);
+                throw new ArgumentException(errorString);
+            }
+
+            File.WriteAllText(fullFileName, 1.ToString());
```

Comment:
Typo?

Reply:
Even if this technically works I'd rather it be "1" or the like.

Reply:
It'll be updated with the next commit.